hack/generate-yamls.sh

```diff
@@ -105,30 +96,9 @@ ko resolve ${KO_YAML_FLAGS} -R -f config/monitoring/tracing/zipkin > "${MONITORI
 # Traces via Zipkin in Memory when ElasticSearch is not installed
 ko resolve ${KO_YAML_FLAGS} -R -f config/monitoring/tracing/zipkin-in-mem >> "${MONITORING_TRACE_ZIPKIN_IN_MEM_YAML}"
 
-echo "Building Release bundles"
-
-# NO_MON is just build and serving
-cp "${BUILD_YAML}" "${RELEASE_NO_MON_YAML}"
-echo "---" >> "${RELEASE_NO_MON_YAML}"
-cat "${SERVING_YAML}" >> "${RELEASE_NO_MON_YAML}"
-echo "---" >> "${RELEASE_NO_MON_YAML}"
-
-# LITE is NO_MON plus "lean" monitoring
-cp "${RELEASE_NO_MON_YAML}" "${RELEASE_LITE_YAML}"
-echo "---" >> "${RELEASE_LITE_YAML}"
-cat "${MONITORING_METRIC_PROMETHEUS_YAML}" >> "${RELEASE_LITE_YAML}"
-echo "---" >> "${RELEASE_LITE_YAML}"
-
-# RELEASE is NO_MON plus full monitoring
-cp "${RELEASE_NO_MON_YAML}" "${RELEASE_YAML}"
-echo "---" >> "${RELEASE_YAML}"
-cat "${MONITORING_YAML}" >> "${RELEASE_YAML}"
-echo "---" >> "${RELEASE_YAML}"
-
 echo "All manifests generated"
 
 # List generated YAML files
 
-ls -1 ${RELEASE_YAML} > ${YAML_LIST_FILE}
-ls -1 ${YAML_OUTPUT_DIR}/*.yaml | grep -v ${RELEASE_YAML} >> ${YAML_LIST_FILE}
+ls -1 ${YAML_OUTPUT_DIR}/*.yaml > ${YAML_LIST_FILE}
```

Comment:
To keep it compatible with the expected output (see line 24), add `${SERVING_YAML}` first to the list, then the rest. Then you don't need to change line 42 (or rely on a hardcoded file name) in `release.sh`.

Reply:
I understand. Now that we don't have bundles anymore, it seems that it's probably not necessary. Let's keep it this way now, so we don't break anything, and revisit this approach soon.